.circleci/config.yml

```diff
@@ -1,56 +1,378 @@
 # Javascript Node CircleCI 2.0 configuration file
 # Check https://circleci.com/docs/2.0/language-javascript/ for more details
-version: 2
-jobs:
-  build:
-    working_directory: ~/typeorm
+version: 2.1
+
+executors:
```

Comment:
Executors are ways of sharing the configuration that a job will need to run. In this case, I've defined an executor for each DB that we test, that way when we run the tests on each version of Node, each of those jobs can simply reference the executor for that DB rather than duplicating it.

Reply:
Note: All of these configs were copied from `docker-compose.yml`